src/api/baseapi.cpp

```diff
@@ -118,10 +118,10 @@ namespace tesseract {
     static void addAvailableLanguages(const STRING &datadir, const STRING &base,
                                       GenericVector<STRING>* langs)
     {
-      const STRING base2 = (base.string()[0] == '\0') ? base : base + "/";
-      const size_t extlen = sizeof(kTrainedDataSuffix);
+        const STRING base2 = (base.string()[0] == '\0') ? base : base + "/";
```

Comment:
You can use C++ comments, and Tesseract uses indentation steps of 2, so at least most of this commit is not needed or even not wanted at all.